app/assets/javascripts/pure_admin/partial_refresh.js

```diff
@@ -4,7 +4,8 @@ PureAdmin.partial_refresh = {
   loadingTimer: {},
 
   init: function() {
-    var matchElem = '.js-partial-refresh';
+    var matchElem = '.pagination, .js-partial-refresh';
```

Comment:
Fair point, I guess the ability there shouldn't be tied to the pagination gem in use either so defining a class isn't ideal.